packages/components/src/styles/prose/prose.stories.ts

```diff
@@ -9,720 +9,186 @@ const { generateTemplate } = storybookTemplate('sd-prose');
-          <p>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie
-            phasellus dui vel id. Velit in sed non orci pellentesque vivamus nunc. At non tortor, sit neque tristique.
-            Facilisis commodo integer hendrerit tortor.
-          </p>
-          <h4>H4 Nisi eu excepteur anim esse</h4>
-          <p>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie
-            phasellus dui vel id. Velit in sed non orci pellentesque vivamus nunc. At non tortor, sit neque tristique.
-            Facilisis commodo integer hendrerit tortor.
-          </p>
-          <h5>H5 Nisi eu excepteur anim esse</h5>
-          <p>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie
-            phasellus dui vel id. Velit in sed non orci pellentesque vivamus nunc. At non tortor, sit neque tristique.
-            Facilisis commodo integer hendrerit tortor.
-          </p>
-        </div>
+          </ul>
+        </li>
+        <li>Unordered list level 1</li>
+        <li>Unordered list level 1</li>
+      </ul>
+      <figure class="sd-media p-4">
+        <img
+          src="./placeholders/images/generic.jpg"
+          alt="A generic placeholder jpg"
+          class="aspect-video object-cover"
+        />
+        <figcaption>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula</figcaption>
+      </figure>
+      <table class="sd-table" style="width: 100%">
+        <thead>
+          <tr>
+            <th class="sd-table-cell">Header</th>
+            <th class="sd-table-cell">Header</th>
+            <th class="sd-table-cell">Header</th>
+          </tr>
+        </thead>
+
+        <tbody>
+          <tr>
+            <td class="sd-table-cell">Cell Content</td>
+            <td class="sd-table-cell">Cell Content</td>
+            <td class="sd-table-cell">Cell Content</td>
+          </tr>
+          <tr>
+            <td class="sd-table-cell">Cell Content</td>
+            <td class="sd-table-cell">Cell Content</td>
+            <td class="sd-table-cell">Cell Content</td>
+          </tr>
+          <tr>
+            <td class="sd-table-cell">Cell Content</td>
+            <td class="sd-table-cell">Cell Content</td>
+            <td class="sd-table-cell">Cell Content</td>
+          </tr>
+          <tr>
+            <td class="sd-table-cell">Cell Content</td>
+            <td class="sd-table-cell">Cell Content</td>
+            <td class="sd-table-cell">Cell Content</td>
+          </tr>
+          <tr>
+            <td class="sd-table-cell">Cell Content</td>
+            <td class="sd-table-cell">Cell Content</td>
+            <td class="sd-table-cell">Cell Content</td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+  `
+};
+
+/**
+ * Use the `sd-prose--full-width` class to make the prose full width.
```

Comment:
I remember we briefly discussed this in the daily. But pointing out class naming inconsistencies. It is the full `sd-prose--full-width` here, but for inverted we used `&--inverted`. 

I, personally, prefer the first style for all, but regardless they should be similar :)

Reply:
Updated